.env.example

```diff
@@ -1,7 +1,9 @@
 SESSION_SECRET=example-secret
 REDIS_URL=//localhost:6379
-API_BASE_URL=http://localhost:4000/api/v1
-API_AUTH_URL=http://localhost:4000/oauth/token
+API_BASE_URL=http://localhost:3000
```

Comment:
worth changing this to `5000`?

Reply:
I actually changed it back to 3000 as it's the default that Rails runs on. But then you'd have to run the frontend on a separate port.

This is more just for guidance as to what env vars to set anyway I suppose so maybe not that important?